content/_code-samples/xrpl-py/get-acct-info.py

```diff
@@ -0,0 +1,35 @@
+# Define the network client
+from xrpl.clients import JsonRpcClient
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+
+# Create a wallet using the testnet faucet: 
+# https://xrpl.org/xrp-testnet-faucet.html
+from xrpl.wallet import generate_faucet_wallet
+test_wallet = generate_faucet_wallet(client)
+
+# Create an account str from the wallet
+test_account = test_wallet.classic_address
+
+# Derive an x-address from the classic address:
+# https://xrpaddress.info/
+from xrpl.core import addresscodec
+test_xaddress = addresscodec.classic_address_to_xaddress(test_account, tag=12345, is_test_network=True)
+print("\nClassic address:\n\n", test_account)
+print("X-address:\n\n", test_xaddress)
+
+
+# Look up info about your account
+from xrpl.models.requests.account_info import AccountInfo
+acct_info = AccountInfo(
+    account=test_account,
+    ledger_index="validated",
```

Comment:
@mDuo13 I changed this to validated but testing code elsewhere, I got an error when trying to use `"validated"`:

```py
>>> acct_info = AccountInfo(
...   account="rBtXmAdEYcno9LWRnAGfT9qBxCeDvuVRZo",
...     ledger_index="validated",
...       queue=True,
...     strict=True,
... )
>>> response = client.request(acct_info)
>>> result = response.result
>>> import json
>>> print(json.dumps(result["account_data"], indent=4, sort_keys=True))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'account_data'
>>> print(result)
{'error': 'invalidParams', 'error_code': 31, 'error_message': 'Invalid parameters.', 'ledger_hash': '78D93327787EFF11BD5AB6507BF81CE5514B422321C798326F954A19E9C33DF6', 'ledger_index': 16234298, 'request': {'account': 'rBtXmAdEYcno9LWRnAGfT9qBxCeDvuVRZo', 'command': 'account_info', 'ledger_index': 'validated', 'queue': True, 'signer_lists': False, 'strict': True}, 'validated': True}
>>> print(response)
Response(status=<ResponseStatus.ERROR: 'error'>, result={'error': 'invalidParams', 'error_code': 31, 'error_message': 'ledgerIndexMalformed', 'request': {'account': 'rBtXmAdEYcno9LWRnAGfT9qBxCeDvuVRZo', 'command': 'account_info', 'ledger_index': 'Validated', 'queue': True, 'signer_lists': False, 'strict': True}}, id=None, type=<ResponseType.RESPONSE: 'response'>)
```